diff -Npur pacman-5.2.1.orig/lib/libalpm/be_sync.c pacman-5.2.1/lib/libalpm/be_sync.c
--- pacman-5.2.1.orig/lib/libalpm/be_sync.c	2019-10-24 15:29:08.000000000 +0200
+++ pacman-5.2.1/lib/libalpm/be_sync.c	2020-01-07 21:44:12.930689147 +0100
@@ -225,8 +225,8 @@ int SYMEXPORT alpm_db_update(int force,
 
 		memset(&payload, 0, sizeof(struct dload_payload));
 
-		/* set hard upper limit of 25MiB */
-		payload.max_size = 25 * 1024 * 1024;
+		/* set hard upper limit of 32MiB */
+		payload.max_size = 32 * 1024 * 1024;
 
 		/* print server + filename into a buffer */
 		len = strlen(server) + strlen(db->treename) + strlen(dbext) + 2;
